### DISCUSSION TITLE
Showing focused table

### DISCUSSION THREAD

**Discussion by richarde-work** [ueberdosis/tiptap#4675] [General Questions]
I'm trying to figure out a way to display a highlight or focus around a table when a user has selected a cell or when the cursor is focused in one of the cells as per below,

![image](https://github.com/ueberdosis/tiptap/assets/122756049/1c9aa856-abac-489b-8045-ae565cb96ccf)

I can easily create the highlight in CSS based on a class but how can I add an appropriate class to the table when the cursor is in a cell or when one or more cells are selected? The `selectedCell` class appears when a user selects a cell but that doesn't help if the cursor is simply in one of the cells since there is no corresponding class for this focus. The `isActive('table')` tells me when any table is active but can't identify a specific table if there is more than one.

Is there something else that I'm missing that might help? Or can someone point me in the right direction of what I need to do here utilising more basic commands to add this functionality?

**Comment by EzhVino** [comment]
Try this Focus extension https://tiptap.dev/docs/editor/extensions/functionality/focus
